_data/calendar.yml

```
@@ -748,7 +748,7 @@
 - title: Geschäftsführende Versammlung
   datum: "2024-03-20"
   uhrzeit: "20.00h"
-  ort: "Flex Office 1, 3.OG, Collective Incubator, Jülicher Str. 209d, 52070 Aachen"
+  ort: "FT103, Melatenerstraße 23-25, 52074 Aachen"
```

```suggestion
  ort: "Raum 2090|U203 (FT103), Melatener Str. 23-25"
```

Schreibweise war in der Vergangenheit ([Z. 717](https://github.com/TechAachen/Webseite-Jekyll/blob/536ff58c2ad3bbda699511c30c95d2b40795dd47/_data/calendar.yml#L717), [Z. 723](https://github.com/TechAachen/Webseite-Jekyll/blob/536ff58c2ad3bbda699511c30c95d2b40795dd47/_data/calendar.yml#L723)) anders.

---

? @TheLeo22